.github/workflows/windows.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           key: ${{ runner.os }}-${{ github.sha }}-bridge
           fail-on-cache-miss: true
       - uses: microsoft/setup-msbuild@v1.1
-      - if: ${{ matrix.compiler == 'MSVC' }}
```

Comment:
Dosn't relate to permisssons. Enclosing strings `${{` and `}}` are simply not needed here.